src/pages/Authentication/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,159 @@
+import { Navbar } from "../../../components/Navbar/Navbar";
+import { Footer } from "../../../components/Footer/Footer";
```

Comment:
You can make use of barrel export, it will be more readable and easy to refactor(if needed).

Reply:
Yes, I will implement this in a separate branch. Thank you